pkg/sbom/scanner/scanner.go

```diff
@@ -125,23 +124,19 @@ func (s *Scanner) start(ctx context.Context) {
 
 				scanContext, cancel := context.WithTimeout(ctx, scanTimeout)
 				createdAt := time.Now()
-				report, err := collector.Scan(scanContext, request.ScanRequest, request.opts)
-				generationDuration := time.Since(createdAt)
-				cancel()
-				if err != nil {
+				log.Info("scanning image %s", request.ID())
```

Comment:
I suppose here it can also be a host 

```suggestion
				log.Info("scanning from %v, id: %s", reflect.TypeOf(request), request.ID())
```

Reply:
I forgot to remove this log but it does not hurt to keep it. Perhaps we could even make it a debug log